src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.pow(getRadius(), 2) * Math.PI;
```

Comment:
```suggestion
        return Math.pow(radius, 2) * Math.PI;
```
we don't need to use getters where we can just use field name
please fix everywhere